### ISSUE TITLE
DNS Name server: ANDI output for error messages

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2658]
### Current Behavior

**The `Name server` and `IP address (IPv4 or IPv6)` screen reader/ANDI output does not account for error states.**

IP address (IPv4 or IPv6)
<img width="1246" alt="Screenshot 2024-08-27 at 9 16 19 AM" src="https://github.com/user-attachments/assets/693a3699-b11e-4e1d-bf18-0ecc6a0208e7">
<img width="704" alt="Screenshot 2024-08-27 at 9 12 50 AM" src="https://github.com/user-attachments/assets/1b7ad288-6874-4c6c-830a-915c8ed67538">

Name server
<img width="817" alt="Screenshot 2024-08-27 at 9 15 09 AM" src="https://github.com/user-attachments/assets/524ad18d-84ca-48c7-84c4-11cb03a9e0c4">
<img width="806" alt="Screenshot 2024-08-27 at 9 15 59 AM" src="https://github.com/user-attachments/assets/98d08959-5bf2-4a50-a3da-a51087f4e1eb">
`Name server` and `IP address (IPv4 or IPv6)` `Delete` buttons
The ANDI output does not specify which name server will be deleted.  
<img width="1207" alt="Screenshot 2024-08-27 at 9 35 13 AM" src="https://github.com/user-attachments/assets/02a09d76-6046-4910-bd1a-fa368b354796">

### Expected Behavior

The ANDI output accounts for all error states for the Name server and IP address fields.
**IP address (IPv4 or IPv6)**
- The ANDI output should be: `IP address (IPv4 or IPv6) Example: 86.124.49.54 or 2001:db8::1234:5678 | Using your domain for a name server requires an IP address`

**Name servers**
Depending on the error the ANDI output should be: 
- `Name server 2 * Example: ns2.example.com | You already entered this name server address. Name server addresses must be unique | required`
- `Name server 2 * Example: ns2.example.com | At least two name servers are required`
- `Name server 2 * Example: ns2.example.com | You must provide a name server to enter an IP address | Required`
- If there are 2 errors: `Name server 2 * Example: ns2.example.com | At least two name servers are required  | You must provide a name server to enter an IP address | required`

**`Name server` and `IP address (IPv4 or IPv6)` `Delete` buttons**
- For Name server 1 the Delete buttons ANDI output should be: `Delete name server 1`
- For Name server 2 the Delete buttons ANDI output should be: `Delete name server 2`
- And so on

### Steps to Reproduce

1. As a domain manager edit the DNS Name servers
2. Identify all error messages in the screenshots above

### Environment

Environment: development
Browser: Chrome

### Prioritization Comments
**WCAG Conformance Level:** 2.1 AA
**WCAG Standard that applies:** 
3.3.1 - Error Identification (Level A) - Critical
3.3.3 - Error Suggestion (Level AA) - Critical
4.1.2 - Name, Role, Value (Level A) - Critical
**Critical Failure if Not Met?:**  Yes
**Prioritization:** Blocker
**Reason:** Ensuring accurate screen reader output for Name Server and IP Address fields, including error states, is essential for accessibility. With the current behavior, users are unable to perceive the screen reader outputs for each error state, including unique messages for each type of error on Name Server and IP Address fields, therefore being a blocker to these users.
